CHANGELOG.md

```diff
@@ -1,11 +1,15 @@
-# Changelog
+# Change log
```

Comment:
Seems to be inhomogeneous. refering to all other change in this diff...

Reply:
For this file it popped out of the generator. Maybe Dennis used older versions for the previous version (or just an outdated template)...
And what do you mean with inhomogeneous? What exactly does not fit?

Reply:
At least in the diff all other occurrences use changelog, not change log